config/stamp/stamp.yml

```diff
@@ -121,6 +121,8 @@ NGINX_SSL_KEY: %%NGINX_SSL_KEY%%
 STORAGE_TYPE: "azure"
 AZURE_STORAGE_PROVIDER: "storages.backends.azure_storage.AzureStorage"
 EDXAPP_DEFAULT_FILE_STORAGE: "django.core.files.storage.FileSystemStorage"
+EDXAPP_AZURE_URL_EXPIRES: %%EDXAPP_AZURE_URL_EXPIRES%%
```

Comment:
Suggest: Rename EDXAPP_AZURE_URL_EXPIRES and EDXAPP_AZURE_URL_EXPIRATION_SECS to indicate that they're specific to ORA2 storage (e.g. EDXAPP_ORA2_STORAGE_EXPIRES and EDXAPP_ORA2_STORAGE_EXPIRATION_SECS)